connectors/src/admin/db.ts

```diff
@@ -42,6 +46,8 @@ import logger from "@connectors/logger/logger";
 
 async function main(): Promise<void> {
   await Connector.sync({ alter: true });
+  await ConfluenceConfigurations.sync({ alter: true });
```

Comment:
Table names are plural but models are singular. Not respected everywhere but that's what we should aim for :pray:

Reply:
Ok was not sure about the standards, but will update.